examples/.isort.cfg

```diff
@@ -4,8 +4,3 @@ multi_line_output = 11
 balanced_wrapping = true
 combine_as_imports = true
 case_sensitive = true
-
-known_third_party = kopf
-
-filter_files = true
-skip_glob = kopf/**
```

Comment:
Can you please clarify what was the reason for removing these settings? Currently, they are intentionally formatted differently — with examples considered as 3rd-party apps, as if they are written not in Kopf's repo.